src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,29 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  selectedTodo: Todo;
+  onClose: () => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ selectedTodo, onClose }) => {
+  const [loading, setLoading] = useState(true);
```

Comment:
`isLoading` for boolean variables: https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig